pym/bob/cmds/archive.py

```diff
@@ -86,11 +88,16 @@ def scan(self, verbose):
                     for l3 in os.listdir(l2):
                         m = self.__archiveSchema.fullmatch(l3)
                         if not m: continue
+                        found = True
                         self.__scan(os.path.join(l2, l3), verbose)
         except OSError as e:
             raise BobError("Error scanning archive: " + str(e))
         finally:
             self.__db.execute("END")
+            if verbose and not found:
+                print("Your archive seems to be empty. "
+                      "Are you running 'bob archive' from within the correct directory?")
```

Comment:
I guess that should go to `stderr` because it's nothing parseable...

Reply:
In fact that's what I had initially... :joy: